runtime/compiler/runtime/ClassUnloadAssumption.cpp

```diff
@@ -717,12 +717,33 @@ void TR_UnloadedClassPicSite::compensate(TR_FrontEnd *, bool isSMP, void *)
             }
 
          }
+      //if LLILF we need to convert to LGFI to get sign extension
+      else if (((*((uint16_t *)cursor) & (uint16_t)0xFF0F) == (uint16_t)0xC00F))
```

Comment:
Can you insert tab here? Also I would put the comment inside the `else if` statement for better readability. 
```suggestion
      else if (((*((uint16_t *)cursor) & (uint16_t)0xFF0F) == (uint16_t)0xC00F))
```